spec/requests/registrations_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Registrations', type: :request do
+  describe 'POST #create' do
+    context 'given the email is NOT already taken' do
```

Comment:
As this is the happy case, should we rename this test to:
```suggestion
    context 'given the email is available' do
```